Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/repair.h

```diff
@@ -1089,15 +1089,15 @@ bool remove_degenerate_edges(TriangleMesh& tmesh)
 //         - `Compare_distance_3` to compute the distance between 2 points
 //         - `Collinear_3` to check whether 3 points are collinear
 //         - `Less_xyz_3` to compare lexicographically two points
-///        - `Equal_3` to check whether 2 points are identical.
-///       For each functor Foo, a function `Foo foo_object()` must be provided.
+//        - `Equal_3` to check whether 2 points are identical.
```

Comment:
Indentation is here before the `-` different, in the current situation no problem but when all lines of the comment block would get `///` again it is a small problem (I don't think this is intended in the future so just a small warning).

Reply:
It doesn't matter because the code for 5.1 is in another PR, where that particular function is properly documented, but thanks for the heads-up.